forms/OptionsetField.php

```diff
@@ -102,4 +102,19 @@ public function validate($validator) {
 	public function ExtraOptions() {
 		return new ArrayList();
 	}
+	
+	public function getAttributes() {
+		$attributes = parent::getAttributes();
+		unset($attributes['name']);
+		if(isset($attributes['required'])) {
+			unset($attributes['required']);
+		}
+		if(isset($attributes['role'])) {
+			unset($attributes['role']);
+		}
+		if(isset($attributes['aria-required'])) {
```

Comment:
How about doing something like: http://stackoverflow.com/questions/34300154/is-aria-required-allowed-on-radio-buttons-or-checkboxes

We'd need to put the options into a fieldset, though.